### PR TITLE
Add p9 scale out simulation build support

### PR DESCRIPTION
**Pull request by ploetzma**
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/675)

### CODE REVIEW

File: openpower/package/p9so_sim-xml/p9so_sim.mk [hunk 1]
```diff
@@ -0,0 +1,78 @@
+################################################################################
+#
+# p9so_sim_xml
+#
+################################################################################
+
+P9SO_SIM_XML_VERSION ?= a89100084400694a585273e64f9696b6f3b4da48
+P9SO_SIM_XML_SITE ?= $(call github,sannerd,p9so_sim-xml,$(P9SO_SIM_XML_VERSION))
```
Comment (ploetzma) [inline openpower/package/p9so_sim-xml/p9so_sim.mk]: Will update these when p9so__sim xml repo is in

**Comment by williamspatrick** [comment]
Why are we doing this?  Is there anyone outside of IBM that has access to this simulator?

**Comment by ploetzma** [comment]
@williamspatrick It sounds like this will be utilized by the HDAT team
@sannerd Is that correct?

**Comment by dcrowell77** [comment]
@stewart-ibm Asked for it I think

**Comment by sannerd** [comment]
@williamspatrick  This is desired to allow easier simulation between various teams.

**Comment by stewart-ibm** [comment]
So, I think the Jenkins tests should build all defconfigs, as otherwise we have situations like this one, where it tests okay, but doesn't actulaly test that the new code builds. @e-liner is this something you can look at?

**Comment by stewart-ibm** [comment]
As for having something internal or external, we had the P9 Mambo defconfig for months before there was ever a public P9 Mambo, so I don't see it as a real problem - but if there's a point where it's no longer going to be maintained or tested, then it should drop.

File: openpower/package/p9so_sim-xml/p9so_sim.mk (hunk 1 above)
Comment (stewart-ibm) [inline openpower/package/p9so_sim-xml/p9so_sim.mk]: Of course, should be an open-power repo before we merge :)

**Comment by stewart-ibm** [comment]
Okay... we should make a call on this.

Is this going to give us anything in the next X few months that the zaius/witherspoon builds do not?

**Comment by ploetzma** [comment]
@sannerd Looking for your input here. Is there a p9so__sim xml repo in the works?

**Pull request closed without merging**